ui/app/AppLayouts/Chat/panels/communities/CommunityMintTokensSettingsPanel.qml

```diff
@@ -220,13 +244,23 @@ SettingsPageLayout {
         id: newTokenView
 
         ColumnLayout {
+            id: _colLayout
+
+            property CollectibleObject collectible: CollectibleObject{}
```

Comment:
Is the default instance needed here? What about `required property CollectibleObject collectible`?

Reply:
Yes, it is needed for the form empty flow.